docs/Configuring a Harmony service.ipynb

```diff
@@ -0,0 +1,577 @@
+    "  data_operation_version: '0.10.0' # The version of the data-operation messaging schema to use\n",
+    "  type:                           # Configuration for service invocation\n",
+    "    name: queue                   # The type of service invocation, either \"queue\" or \"http\"\n",
+    "    params:                       # Parameters specific to the service invocation type\n",
+    "      queue_url: !Env ${BASE_QUEUE_URL}harmony-gdal-queue  # The SQS queue to listen to for requests\n",
+    "  data_url_pattern: '.*'          # An optional (default = .*) regular expression for a substring that desired data URLs should contain\n",
+    "  collections:                    # A list of CMR collection IDs that the service works on\n",
+    "    - C1234-EXAMPLE\n",
+    "  batch_size: 1                   # The number of granules in each batch operation (defaults to 0 which means unlimited)\n",
+    "  maximum_sync_granules: 1        # Optional limit for the maximum number of granules for a request to be handled synchronously. Defaults to 1. Set to 0 to only allow async requests.\n",
+    "  maximum_async_granules: 500     # Optional limit for the maximum number of granules allowed for a single async request. Harmony has a MAX_GRANULE_LIMIT enforced for all services.\n",
+    "  capabilities:                   # Service capabilities\n",
+    "    subsetting:\n",
+    "      bbox: true                  # Can subset by spatial bounding box\n",
+    "      variable: true              # Can subset by UMM-Var variable\n",
+    "      multiple_variable: true     # Can subset multiple variables at once\n",
+    "    output_formats:               # A list of output mime types the service can produce\n",
+    "      - image/tiff\n",
+    "      - image/png\n",
+    "      - image/gif\n",
+    "    reprojection: true            # The service supports reprojection\n",
+    "\n",
+    "- name: harmony/http-example      # An example of configuring the HTTP backend\n",
+    "  type:\n",
+    "    name: http                    # This is an HTTP endpoint\n",
+    "    params:\n",
+    "      url: http://www.example.com/harmony  # URL for the backend service\n",
+    "  # ... And other config (collections / capabilities) as in the above docker example\n",
+    "```\n",
+    "\n",
+    "#### Making a PR:\n",
+    "\n",
+    "Once you have a git branch with a workflow template and the necessary entries in the `config/services.yml` file, you should open a pull request to merge those changes into the NASA Harmony repository. Once merged, the changes will need to be deployed to the specified environments to activate your service. At that point you can begin making HTTP requests to retrieve output from your service via `harmony-py`, a browser, the Python `requests` package, cURL, or other client."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### UMM-Var records:\n",
```

Comment:
This is a great tutorial for UMM-Var. Is this UMM-Powerhouse team aware of this? If not, happy to point them to this. I suspect there could be some extra linkages between this documentation and what they already provide.

Reply:
I hadn't spoke to the UMM-Powerhouse team about this. Extra links to their documentation would be great.